serl_robot_infra/franka_env/envs/relative_env.py

```diff
@@ -94,3 +94,12 @@ def transform_action(self, action: np.ndarray):
         action = np.array(action)  # in case action is a jax read-only array
         action[:6] = self.adjoint_matrix @ action[:6]
         return action
+
+    def transform_action_inv(self, action: np.ndarray):
+        """
```

Comment:
can you add description that this assumes last dimension is gripper open/close?

Reply:
There's actually description in the docstring for the entire wrapper class. It just requires the first 6 dimensions to be translation+rotation, and does not assume anything about gripper dimension. 
```python
class RelativeFrame(gym.Wrapper):
    """
    This wrapper transforms the observation and action to be expressed in the end-effector frame.
    Optionally, it can transform the tcp_pose into a relative frame defined as the reset pose.

    This wrapper is expected to be used on top of the base Franka environment, which has the following
    observation space:
    {
        "state": spaces.Dict(
            {
                "tcp_pose": spaces.Box(-np.inf, np.inf, shape=(7,)), # xyz + quat
                ......
            }
        ),
        ......
    }, and at least 6 DoF action space with (x, y, z, rx, ry, rz, ...)
    """
```

Reply:
Ok, then what's the assumption for the 7th>?

Reply:
Doesn't assume anything for the 7th. It can also be non-existant. All the code only indices the first 6 dimensions `action[:6]`

Reply:
then can we add a simple description of the expected 7th dimension? e.g., discrete/continuous gripper status